.jenkins/test_config_template.yaml

```diff
@@ -0,0 +1,185 @@
+env_aliases:
+  runtime:
+    eager:
+      PT_HPU_LAZY_MODE: '0'
+    lazy:
+      PT_HPU_LAZY_MODE: '1'
+    default: lazy
+  engine:
+    v0:
+      VLLM_USE_V1: '0'
+    v1:
+      VLLM_USE_V1: '1'
+    default: v0
+  warmup:
+    skip_warmup:
+      VLLM_SKIP_WARMUP: 'true'
+    do_warmup:
+      VLLM_SKIP_WARMUP: 'false'
+    default: skip_warmup
+
+# Define reusable anchors globally
+name_template: &name_template '{{ runtime }}_{{ name }}_{{ engine }}_{{ flavor }}'
+env:
+  default_env: &default_env {runtime: [eager, lazy], engine: [v0, v1], warmup: [skip_warmup]}
+  v0_env: &v0_env {runtime: [eager, lazy], engine: [v0], warmup: [skip_warmup]}
+  v1_env: &v1_env {runtime: [eager, lazy], engine: [v1], warmup: [skip_warmup]}
```

Comment:
How can I make here env with different flags only for eager (t.compile) as current compile version has) ?
It seems to be missing mechanism to apply one env_alias only if other env_alias is just used in generation of output  yaml